scripts/bash_completion/optirun

```diff
@@ -54,7 +54,8 @@ _optirun() {
           *)
```

Comment:
8eb1df2945e7ca33028004eed58d7d735cf54c9c should fix it, and while at it I also split line 19 so that there aren't any lines longer than 80 characters.
thanks for the feedback :+1: